app/Tasks/Status.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Tasks;
+
+use App\Builder\Docker;
+use App\Builder\DockerCompose;
+use App\Builder\Escaped;
+
+class Status extends Task
+{
+    /** @var bool $quite */
+    protected $quite = false;
+
+    public function run(...$args): int
+    {
+        $start = Start::make($this->command);
+
+        $chain = [
+            [$start, 'handleNetwork'],
+            [$this, 'displayServices'],
+        ];
+
+        return $this->runCallables($chain);
+    }
+
+    public function quite(bool $quite) : self
+    {
+        $this->quite = $quite;
+
+        return $this;
+    }
+
+    public function displayServices() : int
+    {
+        $lines = $this->getOutputLines(DockerCompose::make('ps', '--services'));
+
+        $services = collect($lines)
+            ->sort()
+            ->map(function ($service) {
+                $info = [];
+                $isRunning = $this->isRunning($service, $info);
+
+                return [
+                    $service,
+                    $isRunning
+                        ? '<fg=white;bg=green;options=bold>Running</>'
```

Comment:
this styling looks useful, maybe we can create a helper class for things like this?